ecpy_pulses/manifest.enaml

```diff
@@ -0,0 +1,678 @@
+                    dependencies[el] = cls
+
+
+    Extension:
+        id = 'workspace'
+        point = u'enaml.workbench.ui.workspaces'
+        factory = workspace_factory
+
+    Extension:
+        id = 'commands'
+        point = 'enaml.workbench.core.commands'
+#        Command:
+#            id = 'ecpy.instr_manager.show_manager'
+#            description = SHOW_MANAGER_DESCRIPTION
+#            handler = show_manager
+        Command:
+            id = 'ecpy.pulses.show_starting_errors'
+            description = SHOW_ERRORS_DESCRIPTION
+            handler = show_starting_errors
+        Command:
+            id = 'ecpy.pulses.get_sequences_infos'
+            description = GET_SEQUENCES_INFOS_DESCRIPTION
+            handler = make_handler(PLUGIN_ID, 'get_sequences_infos')
+        Command:
+            id = 'ecpy.pulses.get_sequence_infos'
+            description = GET_SEQUENCE_INFOS_DESCRIPTION
+            handler = make_handler(PLUGIN_ID, 'get_sequence_infos')
+        Command:
+            id = 'ecpy.pulses.get_sequences'
+            description = GET_SEQUENCES_DESCRIPTION
+            handler = make_handler(PLUGIN_ID, 'get_sequences')
+        Command:
+            id = 'ecpy.pulses.get_sequence'
+            description = GET_SEQUENCE_DESCRIPTION
+            handler = make_handler(PLUGIN_ID, 'get_sequence')
+        Command:
+            id = 'ecpy.pulses.get_items_infos'
+            description = GET_ITEMS_INFOS_DESCRIPTION
+            handler = make_handler(PLUGIN_ID, 'get_items_infos')
+        Command:
+            id = 'ecpy.pulses.get_item_infos'
+            description = GET_ITEM_INFOS_DESCRIPTION
+            handler = make_handler(PLUGIN_ID, 'get_item_infos')
+        Command:
+            id = 'ecpy.pulses.get_contexts_infos'
+            description = GET_CONTEXTS_INFOS_DESCRIPTION
+            handler = make_handler(PLUGIN_ID, 'get_contexts_infos')
+        Command:
+            id = 'ecpy.pulses.get_context_infos'
+            description = GET_CONTEXT_INFOS_DESCRIPTION
+            handler = make_handler(PLUGIN_ID,'get_context_infos')
+        Command:
+            id = 'ecpy.pulses.get_shapes_infos'
+            description = GET_SHAPES_INFOS_DESCRIPTION
+            handler = make_handler(PLUGIN_ID,'get_shapes_infos')
+        Command:
+            id = 'ecpy.pulses.get_shape_infos'
+            description = GET_SHAPE_INFOS_DESCRIPTION
+            handler = make_handler(PLUGIN_ID,'get_shape_infos')
+        Command:
+            id = 'ecpy.pulses.build_sequence'
+            description = BUILD_SEQUENCE_DESCRIPTION
+            handler = build_sequence
+        Command:
+            id = 'ecpy.pulses.build_context'
+            description = BUILD_CONTEXT_DESCRIPTION
+            handler = build_context
+
+    Extension:
+        id = 'menu'
+        point = 'enaml.workbench.ui.actions'
+        MenuItem:
+            path = '/tools/pulses'
+            label = 'Pulses'
+#            after = 'tasks'
```

Comment:
My point was to remove it fully as there is no longer a task entry in 
that menu


Le 08/06/2016 à 17:39, Filippo Vicentini a écrit :
>
> In ecpy_pulses/manifest.enaml 
> <https://github.com/Ecpy/ecpy_pulses/pull/2#discussion_r66280299>:
>
> > +        Command:
> > +            id = 'ecpy.pulses.build_sequence'
> > +            description = BUILD_SEQUENCE_DESCRIPTION
> > +            handler = build_sequence
> > +        Command:
> > +            id = 'ecpy.pulses.build_context'
> > +            description = BUILD_CONTEXT_DESCRIPTION
> > +            handler = build_context
> > +
> > +    Extension:
> > +        id = 'menu'
> > +        point = 'enaml.workbench.ui.actions'
> > +        MenuItem:
> > +            path = '/tools/pulses'
> > +            label = 'Pulses'
> > +#            after = 'tasks'
>
> The comments are there as a todo. If you remove them the application 
> does not even start up, logging the following trace:
>
> (I guess it's something ecpy related)
>
> |2016-06-08` 17:36:29,797 | MainProcess | INFO | Log system started. 
> 2016-06-08 17:36:30,176 | MainProcess | INFO | The Ecpy_Pulses plugin 
> has been loaded. ... 2016-06-08 17:36:30,467 | MainProcess | CRITICAL 
> | raise ValueError(msg % (node.path, after)) 2016-06-08 17:36:30,467 | 
> MainProcess | CRITICAL | ValueError 2016-06-08 17:36:30,468 | 
> MainProcess | CRITICAL | : 2016-06-08 17:36:30,468 | MainProcess | 
> CRITICAL | item '/tools/pulses' has invalid `after` reference 'tasks' |
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Ecpy/ecpy_pulses/pull/2/files/7d5413953398a75cca8093d6814de0543cc78382#r66280299>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AERizBu0WhKu9aWgxwSABSvmFcNHrszJks5qJuIfgaJpZM4Itp11>.
>